### DISCUSSION TITLE
Error: Cask 'isimulator' is unavailable: No Cask with this name exists.

### DISCUSSION THREAD

**Discussion by tubbylumpkinz** [Homebrew/discussions#3687] [Getting started]
Just installed homebrew on chromebook via the Linux Dev Tools on ChromeOS. Not sure how to fix and cant seem to find someone with the same issue. 

LOG
____________

brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!
Warning: Your Homebrew's prefix is not /home/linuxbrew/.linuxbrew.
Some of Homebrew's bottles (binary packages) can only be used with the default
prefix (/home/linuxbrew/.linuxbrew).
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
unsupported configuration.
donwellington666@penguin:~$ brew install --cask isimulator
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> New Formulae
dynein                                     python-typing-extensions
Error: Cask 'isimulator' is unavailable: No Cask with this name exists.

BREW CONFIG

brew config
HOMEBREW_VERSION: 3.6.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: b8b195cc64a29595797651720ebb2ea09affb682
Last commit: 6 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 6fc08d005cfe92ca8453912306fa5704c90374b2
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /home/donwellington666/.linuxbrew
HOMEBREW_REPOSITORY: /home/donwellington666/.linuxbrew/Homebrew
HOMEBREW_CELLAR: /home/donwellington666/.linuxbrew/Cellar
HOMEBREW_BROWSER: /usr/bin/garcon-url-handler
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.8 => /home/donwellington666/.linuxbrew/Homebrew/Library/Homebrew/
vendor/portable-ruby/2.6.8_1/bin/ruby
CPU: quad-core 64-bit unknown_0x6_0x7a
Clang: N/A
Git: 2.30.2 => /bin/git
Curl: 7.74.0 => /bin/curl
Kernel: Linux 5.10.142-19731-gd1d5aaa631a4 x86_64 GNU/Linux
xorg: N/A

**Comment by carlocab** [comment]
Casks are macOS-only. The error message is meant to be better here, but no one's gotten around to fixing it yet. Apologies for the inconvenience!
